pages.ja/common/apktool.md

```diff
@@ -0,0 +1,16 @@
+# apktool
+
+> APK ファイルの解析ツールです。
+> 詳しくはこちら: <https://ibotpeaches.github.io/Apktool/>
```

Comment:
```suggestion
> 詳しくはこちら: <https://ibotpeaches.github.io/Apktool/>.
```

Reply:
Thanks! I've added the period to end of the description.